packages/widgets/src/components/TradingScoreboard/TradingScoreboard.tsx

```diff
@@ -0,0 +1,40 @@
+import { TradingInfoCard } from '@components/TradingInfoCard';
+import TradingScoreboardBase from './TradingScoreboardBase';
+import { ITradingScoreboardProps } from './types';
+
+const TradingScoreboard = ({
+  items,
+  onClick,
+  renderCard,
+  ...props
+}: ITradingScoreboardProps) => {
+  return (
+    <div>
+      <TradingScoreboardBase {...props} data-testid="score-board">
+        {items.map((item, index) => {
+          return (
+            <div key={index}>
+              {typeof renderCard === 'function' ? (
+                renderCard(item, onClick)
+              ) : (
+                <TradingInfoCard
```

Comment:
I think we both missed that the `<TradingInfoCard />` can be a link, not just a button. 
Let's discuss on Monday how we can integrate React Router to the card. 

https://reactrouter.com/en/main/components/link

Reply:
Done